Makefile

```diff
@@ -11,6 +11,7 @@ endif
 
 VERSION = $(shell git describe --dirty --tags --always)
 GIT_COMMIT = $(shell git rev-parse HEAD)
+K8S_VERSION = v1.16.3
```

Comment:
This needs to be piped through to be used [here](https://github.com/operator-framework/operator-sdk/blob/b160c23bf1d89fea4caa3d070d07dc98616b5e16/hack/ci/setup-k8s.sh#L9). And we need to remove the setting in the line above that so that we get it from an argument instead.

That way we don't have a possibility of testing against one version and printing another version.

Maybe create a new make target?
```make
setup-k8s:
	hack/ci/setup-k8s.sh ${K8S_VERSION}
```

And then change [.travis.yml](https://github.com/operator-framework/operator-sdk/blob/b160c23bf1d89fea4caa3d070d07dc98616b5e16/.travis.yml#L52) to use it:
```
make setup-k8s
```

We probably don't need the `CLUSTER` environment variable anymore since we only ever set it to `k8s` in `.travis.yml` in that one section.

Reply:
@theishshah ping